conanfile.py

```diff
@@ -81,9 +70,6 @@ def requirements(self):
                 self.requires(req)
 
     def generate(self):
-        cmake = CMakeDeps(self)
```

Comment:
See comment above. I would prefer to have all the  generators in one place

Reply:
https://github.com/Ultimaker/libnest2d/pull/8/commits/ca60db19bc503aedec491075282fec1eb0b7882c